expression/column.go

```diff
@@ -722,3 +733,20 @@ func GcColumnExprIsTidbShard(virtualExpr Expression) bool {
 
 	return true
 }
+
+const emptyColumnSize = int64(unsafe.Sizeof(Column{}))
+
+// MemoryUsage return the memory usage of Column
+func (col *Column) MemoryUsage() (sum int64) {
+	if col == nil {
+		return
+	}
+
+	sum = emptyColumnSize + col.RetType.MemoryUsage() + int64(cap(col.hashcode))*int64(unsafe.Sizeof(*new(byte))) +
```

Comment:
Is `int64(cap(col.hashcode))` enough? If I remember correctly, the size of 1 byte is always 1?